pkg/kubelet/dockertools/labels.go

```diff
@@ -34,20 +35,28 @@ import (
 //  * label filters (maybe in the future)
 
 const (
-	kubernetesPodNameLabel                   = "io.kubernetes.pod.name"
-	kubernetesPodNamespaceLabel              = "io.kubernetes.pod.namespace"
-	kubernetesPodUIDLabel                    = "io.kubernetes.pod.uid"
-	kubernetesPodDeletionGracePeriodLabel    = "io.kubernetes.pod.deletionGracePeriod"
-	kubernetesPodTerminationGracePeriodLabel = "io.kubernetes.pod.terminationGracePeriod"
-
-	kubernetesContainerNameLabel                   = "io.kubernetes.container.name"
-	kubernetesContainerHashLabel                   = "io.kubernetes.container.hash"
-	kubernetesContainerRestartCountLabel           = "io.kubernetes.container.restartCount"
-	kubernetesContainerTerminationMessagePathLabel = "io.kubernetes.container.terminationMessagePath"
-	kubernetesContainerPreStopHandlerLabel         = "io.kubernetes.container.preStopHandler"
+	podNameLabel                   = "io.kubernetes.pod.name"
```

Comment:
+1. If these labels were to be applied across more than one component,
thats when it gets ugly. Since that is not the case as of now, a version
label is fine for now.

On Mon, Apr 4, 2016 at 3:49 PM, Yu-Ju Hong <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/labels.go
> <https://github.com/kubernetes/kubernetes/pull/23725#discussion_r58462211>
> :
>
> > @@ -34,20 +35,28 @@ import (
> >  //  * label filters (maybe in the future)
> >
> >  const (
> > -	kubernetesPodNameLabel                   = "io.kubernetes.pod.name"
> > -	kubernetesPodNamespaceLabel              = "io.kubernetes.pod.namespace"
> > -	kubernetesPodUIDLabel                    = "io.kubernetes.pod.uid"
> > -	kubernetesPodDeletionGracePeriodLabel    = "io.kubernetes.pod.deletionGracePeriod"
> > -	kubernetesPodTerminationGracePeriodLabel = "io.kubernetes.pod.terminationGracePeriod"
> > -
> > -	kubernetesContainerNameLabel                   = "io.kubernetes.container.name"
> > -	kubernetesContainerHashLabel                   = "io.kubernetes.container.hash"
> > -	kubernetesContainerRestartCountLabel           = "io.kubernetes.container.restartCount"
> > -	kubernetesContainerTerminationMessagePathLabel = "io.kubernetes.container.terminationMessagePath"
> > -	kubernetesContainerPreStopHandlerLabel         = "io.kubernetes.container.preStopHandler"
> > +	podNameLabel                   = "io.kubernetes.pod.name"
>
> docker doesn't support changing labels on a container yet. If it does, I
> think it makes more sense to re-label the container with a consistent
> version (vs. mixing versions). just my 2 cents.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23725/files/4dcd55dcfd6d48a5f6a1ef81ab1acd2984b540d1#r58462211>
>